pkg/cmd/options.go

```diff
@@ -49,11 +49,11 @@ func ControllerSwitchOptions() *controllercmd.SwitchOptions {
 }
 
 // WebhookSwitchOptions are the webhookcmd.SwitchOptions for the provider webhooks.
-func WebhookSwitchOptions(gardenerVersion *string) *webhookcmd.SwitchOptions {
+func WebhookSwitchOptions(_ *string) *webhookcmd.SwitchOptions {
```

Comment:
will also be overwritten by https://github.com/gardener/gardener-extension-provider-openstack/pull/797/files#diff-89748d82aea4d0150921b534dbd18153a398e6659aa5c285b3550588e906b380R54 again, but should not pose a big problem. either PR needs rebase then anyway.

Reply:
I now rebased after the merge of #797.